plugin/common/common.cuh

```diff
@@ -326,7 +326,7 @@ template <typename T, unsigned TPB>
 __device__ inline void scaledSoftmax(
     const int ld, const int lastValid, const float rsqrtHeadSize, const T* input, T* output)
 {
-
+    
```

Comment:
could you also remove the blanks in this line? thanks

Reply:
removed.